chameleonultragui/lib/bridge/chameleon.dart

```diff
@@ -732,7 +745,13 @@ class ChameleonCom {
 
   Future<int> getActiveSlot() async {
     // get the selected slot on the device, 0-7 (8 slots)
-    return (await sendCmdSync(ChameleonCommand.getActiveSlot, 0x00))!.data[0];
+    var resp = (await sendCmdSync(ChameleonCommand.getActiveSlot, 0x00));
```

Comment:
Never make fallbacks in bridge, create them only in GUI

Reply:
What do you suggest to return then, null or -1?

Reply:
exception

Reply:
@Foxushka Would you be ok with throwing this exception in sendCmdSync whenever `resp.status == ChameleonStatus.notImplemented` or `ChameleonStatus.invalidCommand`?